helm/envoy-gateway-default-configuration/values.yaml

```diff
@@ -7,6 +8,9 @@ project:
 
 image:
   registry: quay.io
-  name: giantswarm/{APP-NAME}
+  name: giantswarm/envoy-gateway-default-configuration
   tag: v0.0.1
   pullPolicy: IfNotPresent
+
+gateway:
+  host: test.gaws.gigantic.io
```

Comment:
not sure we should have a default here, that won't work right?

Reply:
we have a dummy value like in IC